vm/actor/src/builtin/miner/mod.rs

```diff
@@ -1315,278 +1469,18 @@ impl Actor {
                 MAX_MINER_PROVE_COMMITS_PER_EPOCH
             );
         }
-
-        // get network stats from other actors
-        let reward_stats = request_current_epoch_block_reward(rt)?;
-        let power_total = request_current_total_power(rt)?;
-        let circulating_supply = rt.total_fil_circ_supply()?;
-
-        // 1. Activate deals, skipping pre-commits with invalid deals.
-        //    - calls the market actor.
-        // 2. Reschedule replacement sector expiration.
-        //    - loads and saves sectors
-        //    - loads and saves deadlines/partitions
-        // 3. Add new sectors.
-        //    - loads and saves sectors.
-        //    - loads and saves deadlines/partitions
-        //
-        // Ideally, we'd combine some of these operations, but at least we have
-        // a constant number of them.
-
-        let state = rt.state()?;
-        let info = get_miner_info(rt.store(), &state)?;
-
-        //
-        // Activate storage deals.
-        //
-
+        let st: State = rt.state()?;
+        let store = rt.store();
         // This skips missing pre-commits.
-        let precommitted_sectors = state
-            .find_precommitted_sectors(rt.store(), &params.sectors)
+        let precommited_sectors = st
```

Comment:
Did you mean to rename `state` here? Coz, from potential future me to potential future you, please do what you can to make it easier to read...

Reply:
I see, that's what they called it here:
https://github.com/filecoin-project/specs-actors/pull/1381/files#diff-d807293f55563b410c2d07719d2f38727016f41c6a17df8635500abc4070f74cR939-R943

Resolving.